packages/stryker-elements/src/lib/atoms/loader.ts

```diff
@@ -1,22 +1,67 @@
 import { html } from 'lit';
-import { classMap } from 'lit/directives/class-map.js';
 import { customElement, property } from 'lit/decorators.js';
-
+import './spinner';
 import { BaseElement } from '../base-element';
+import { classMap } from 'lit/directives/class-map.js';
 
 @customElement('sme-loader')
 export class Loader extends BaseElement {
+  private _doneWithLoading = false;
+
   @property({ type: Boolean, reflect: true })
-  doneWithLoading = false;
+  get doneWithLoading() {
+    return this._doneWithLoading;
+  }
+
+  set doneWithLoading(value: boolean) {
+    const oldValue = this._doneWithLoading;
+    this._doneWithLoading = value;
+    this.requestUpdate('doneWithLoading', oldValue);
+    this.onDoneWithLoadingChanged(value);
+  }
```

Comment:
I'm not entirely sure if this is the correct way to go about it, but I understand why it's done this way.